tests/src/Traits/ClientTestTrait.php

```diff
@@ -39,4 +39,16 @@ protected function getTestingClient(array $configuration = [], array $responseQu
             $configuration
         );
     }
+
+    /**
+     * @param ClientInterface $client
+     * @return mixed
+     */
+    protected function getClientContainerProperty(ClientInterface $client)
```

Comment:
I think it make sense to simplify to `getClientContainer()`. That "property" sounds weird.